ArmPlatformPkg/Library/PL111Lcd/PL111Lcd.c

```diff
@@ -117,7 +117,7 @@ LcdSetMode (
   }
 
   // Disable the CLCD_LcdEn bit
-  MmioAnd32 (PL111_REG_LCD_CONTROL, ~PL111_CTRL_LCD_EN);
+  MmioAnd32 (PL111_REG_LCD_CONTROL, (UINT32) ~PL111_CTRL_LCD_EN);

```

Comment:
Instead of a cast, maybe we should change the PL111_CTRL_LCD_EN in the header file to:
`#define PL111_CTRL_LCD_EN  1U`

Reply:
done